plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss

```diff
@@ -219,7 +220,7 @@
 			&:focus {
 				color: #171717;
 				background: #fcfcfc;
-				border: 1.5px solid var(--wp-admin-theme-color);
+				border-color: var(--wp-admin-theme-color);
```

Comment:
Fix for the jumpy border:

![8rnof2.gif](https://github.com/woocommerce/woocommerce/assets/3747241/6294ed46-2e5c-4da7-b374-d385315e49d5)